pdns/pdns_nod.cc

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
It looks like this file and the corresponding .hh (and symlinks) are not in use?